platform-docs-migration/README.md

```diff
@@ -0,0 +1,69 @@
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+summary: Revamp our docs to describe the Dev Platform product, considering the new Cluster API (CAPI) architecture. Temporarily create a second domain and create the new content from scratch instead of modifying existing docs. Start with content skeleton and also copy some existing content that can go directly into the new site.
+---
+
+# Revamp our docs
+
+## Intro
+
+As part of the [Documentation Overhaul](https://github.com/giantswarm/giantswarm/issues/29503) that Horizon is working on during the first quarter of 2024 there was an idea of how to transition our current docs to a desired state where the new Dev Platform product is described correctly and precisely.
+
+## State of the documentation
+
+Our [current documentation hub](https://docs.giantswarm.io) is an ongoing effort to maintain old Vintage entries that are no longer valid in our new product, and at the same time adequate to the new Dev Platform and CAPI concept.
+
+Today we have several work streams trying to achieve that goal:
+
+1) Horizon and SIG docs maintaining a coherent docs structure that follows new product developments.
+2) Teams updating doc entries for out-of-date documents. Most of the time they don`t have the time to rewrite the content to make it Platform/CAPI ready, so they stay out of date.
+3) Teams, KaaS and platform, adding new content that is only working for the new implementations (CAPI).
+
+## Goals
+
+The described scenario is not working and ideally we find an alternative that has these goals:
+
+- Have a good documentation portal that does not have entries out of date. Only new implementation (since we have completely deprecated Vintage).
+- Stop trying to bring current articles to life, most of them are no longer valid in the new implementation. Just add what is relevant and possibly identify spots to fill by teams.
+- Have a period for transition where everyone can look for content in Vintage and Dev Platform docs. Customers or support still have access to Vintage docs but new docs are also available.
+
+## Process
+
+Instead of creating a huge Pull Request in [our docs repository](https://github.com/giantswarm/docs) where we work in parallel (accumulating a lot of changes and configuration drifts), I propose to create a new clean repository for the new docs. This new repo can be deployed in a temporal (sub)domain where we can visualize the changes till we reach a point of satisfaction and furthermore, customers are migrated from Vintage product.
+
+**But why?**
+
+- In the new repository we can create a structure according to the new necessities without having to transition from existing content.
+- We don`t need to maintain aliases or links to old entries or sections.
+- It is way easier to organize the work between teams to add needed minimal content for every section.
+- We can work in iterations.
+- Both docs sites will be publicly available.
```

Comment:
Maybe we can consider deployment to the same domain, but with a clear selection between 2 "contents". This way, all URLs can remain intact and we still start from scratch, rendering content from a different directory for CAPI stuff.

[Example from competitor that offers a version selection dropdown](https://docs.d2iq.com/dkp/2.7/)

This needs not block the RFC, but maybe we can write this down as technical alternative to shortly try.